bundles/CoreBundle/src/Command/SearchBackendReindexCommand.php

```diff
@@ -64,6 +75,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     DataObject::OBJECT_TYPE_VARIANT,
                 ]);
             }
+            // Load the data that has already been processed
+            if (!$isFull && $ids = $db->fetchFirstColumn("SELECT id FROM search_backend_data WHERE maintype = '". $type ."' ")) {
+                $list->setCondition($type === 'object' ? 'o_id' : 'id' .' NOT IN (:oids)', ['oids' => $ids]);
```

Comment:
Maybe this is wrong?
Should this resolve to `'o_id'` or `'o_id NOT IN (:oids)'`?
I think it should be changed to:
`$list->setCondition(($type === 'object' ? 'o_id' : 'id') .' NOT IN (:oids)', ['oids' => $ids]);`

We did this and without the proposed change he restarted all objects every time.

Reply:
As this is for Pimcore 11... wasn't the `o_`-prefix recently removed?